corehq/apps/celery/models.py

```diff
@@ -4,12 +4,14 @@
 class TaskRecordState(models.TextChoices):
     # the following are meant to replicate celery task states
     # https://docs.celeryq.dev/en/latest/reference/celery.states.html
+    FAILURE = "FAILURE", "Failure"
     PENDING = "PENDING", "Pending"  # task was created, but not yet sent to the broker
+    RECEIVED = "RECEIVED", "Received"  # task was received by a worker, not a broker
+    REJECTED = "REJECTED", "Rejected"  # not in docs, but set when task cannot be processed by worker
```

Comment:
Is this a completed state? Should we just delete (or not create) the record if it's rejected?